advanced_source/ddp_pipeline.py

```diff
@@ -1,35 +1,36 @@
 """
-Training Transformer models using Distributed Data Parallel and Pipeline Parallelism
-====================================================================================
+분산 데이터 병렬 처리와 파이프라인 병렬화를 사용한 트랜스포머 모델 학습
+=======================================================================
 
 **Author**: `Pritam Damania <https://github.com/pritamdamania87>`_
+  **번역**: `백선희 <https://github.com/spongebob03>`_
+  
+이 튜토리얼은 `분산 데이터 병렬처리(Distributed Data Parallel) <https://pytorch.org/docs/stable/generated/torch.nn.parallel.DistributedDataParallel.html>`__ 와
+`파이프라인 병렬화 <https://pytorch.org/docs/stable/pipeline.html>`__
+를 사용하여 여러 GPU에 걸친 거대한 트랜스포머(transformer) 모델을 어떻게 학습시키는지 보여줍니다.
+이번 튜토리얼은 `NN.TRANSFORMER 와 TORCHTEXT 로 시퀀스-투-시퀀스(SEQUENCE-TO-SEQUENCE) 모델링하기 <https://tutorials.pytorch.kr/beginner/transformer_tutorial.html>`__ 의
+확장판이며 파이프라인 병렬화가 어떻게 트랜스포머 모델 학습에 쓰이는지 증명하기 위해 이전 튜토리얼에서의
```

Comment:
`분산 데이터 병럴 처리` 라는 말이 빠졌다는 생각이 듭니다.

```suggestion
확장판이며 분산 데이터 병럴 처리와 파이프라인 병렬화가 어떻게 트랜스포머 모델 학습에 쓰이는지 증명하기 위해 이전 튜토리얼에서의
```